py/mpconfig.h

```diff
@@ -1393,6 +1393,11 @@ typedef double mp_float_t;
 #define MICROPY_PY_MATH_POW_FIX_NAN (0)
 #endif
 
+// Whether to provide fix for gamma(-inf) to raise ValueError
+#ifndef MICROPY_PY_MATH_GAMMA_FIX_NEG_INF
+#define MICROPY_PY_MATH_GAMMA_FIX_NEG_INF (0)
```

Comment:
I suggest to call it `MICROPY_PY_MATH_GAMMA_FIX_NEGINF` (no underscore near the end) to match other names, like `NEGZERO`.